### ISSUE TITLE
Any updates to parity 1.10?

### ISSUE THREAD

**Issue by vladyslav2** [paukstis/freebsd_parity#2]
Do you plan to support this repo in future?

**Comment by paukstis** [comment]
yes, not sure how it will go with 1.10, but I'll definitively revisit it, would be nice though if paritytech made it less Linux dependant, like optional usb hardware.

**Comment by vladyslav2** [comment]
@paukstis thank you
I'll be glad to join you, do some tests on this weekend.

> less Linux dependant, like optional usb hardware.
Should we create one more issue in they repo?
